### ISSUE TITLE
Comments by Radu Făciu

### ISSUE THREAD

**Issue by lgarron** [thewca/wca-regulations#66]
Perhaps some of these should still be addressed.

From an email by Radu Făciu to the WRC, 2012-09-23

> One of my main objectives is that the new Regulations should be made as "small" and compact as possible.

Regulations
- [ ] "1a) I think the "score taker" should be totally removed from the regulations as it's always (almost) the same with the judge..."
- [ ] "1g1) What is a "score taker"? It is not defined anywhere and I think in 99% of the cases the judge is the same with the score taker...so if you want to keep this article, I think it should be replaced by Judge."
- [ ] "1c5) I don't see a point to have this article mentioned....I mean it's obvious. Maybe it sound childish, but I think we are just making the Regulations longer without adding anything with this article and a few others."
- [ ] "2t) Same as above. I think this article is useless...I don't see its purpose."
- [x] "3c) "color scheme with one colour per face" ....I think it should be like "one DIFFERENT colour per each face" as even a fully white stickered puzzle has one colour per face."
- [ ] "3f) I am not a megaminx competitor, but I always was asking myself why tiled Megaminxes like the Meffert's ones are allowed, as you can easily see more of the other side."
- [ ] "11e) It is not clear for me how many extra attempts are possible? Maybe we should specify?"
- [ ] "A3b) What happens if the judge forgets to reset the timer?"
- [ ] "A5b) What if the competitor drops the cube on the floor? We cannot give a DNF because he used the floor instead of the surface."

Guidelines
- [x] "1c4) I think this "recommendation" can/should be removed.... I don't see its point, as a correction is observed the Delegate or whoever, will notify the Board as soon as he can. So I don't see why to mention 1 week...what if it is 2 weeks?"
- [ ] "4d) I don't agree with this point. I think Black should be treated as White ok white puzzles where White is replaced. I think most of us got used to this already and it's quite obvious why. Why change this now? Maybe I didn't understand this point."

**Comment by pedrosino** [comment]
My comments didn't get saved :(
- I agree about the score taker thing. No real need to mention it.
- I do think 2t should stay, though.
- Also agree with him about 3c.
- 11e - I believe there's not a limit to extra attempts.
- A3b - then the solver should get an extra attempt
- A5b - the competitor is not "receiving assistance" from the floor, so I don't see a problem.

## Guidelines
- 1c4) Agreed here. Corrections may happen more than 1 week after the comp, after results are posted and people are checking videos or whatever.
- 4d) I like it the way it is. Much easier to get it right. What about a black cube with black stickers? Or white cube with black instead of yellow? Many possible cases, don't think we should cover each one individually

**Comment by lgarron** [comment]
- Score taker: This is the person who enters stuff on CubeComps. They are important, and distinct from the judge. But I agree that the Regulations don't describe the role very clearly. I've created issue #120 for this.
- 1c5) and 2t)
- 3f) Issue #50 exists for this.
- 11e) As many extra attempts as needed. If there are unreasonably many, the Delegate can handle it. (I don't think this comes up much, and I think it's better to focus on the Regulations being _relevant_ than _short_.
- A3b) I know what we usually do, but I don't have a good answer for what we _should_ do. I've created issue #121.
- A5b) This could be more explicit, but it doesn't cause problems in practice.
- 1c4) Already addressed in 2013 by moving it to Guideline 1c4+.
- 4d) As Pedro said, this rule is very clear and covers every case.
